charts/streams-app/templates/deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
       release: {{ .Release.Name }}
   template:
     metadata:
-    {{- if or (or (or .Values.awsRole .Values.prometheus.jmx.enabled) .Values.autoscaling.consumerGroup) .Values.annotations }}
+    {{- if or (or (or .Values.awsRole .Values.prometheus.jmx.enabled) .Values.autoscaling.consumerGroup) .Values.podAnnotations }}
```

Comment:
remove the consumerGroup condition along with the next comment

Reply:
> remove the consumerGroup condition along with the next comment

I changed it, but here I am not 100% sure, if I got the condition right

Reply:
Looks good! (and much much better with the single or)